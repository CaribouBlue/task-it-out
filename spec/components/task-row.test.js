import React from 'react';
import renderer from 'react-test-renderer';
import TaskRow from 'src/components/task-row';
import MockTasks from 'mock-data/tasks';

let consoleErrorSpy = null
beforeEach(() => {
  consoleErrorSpy = jest.spyOn(global.console, 'error')
})

afterEach(() => {
  expect(consoleErrorSpy).not.toHaveBeenCalled();
})

test('TaskRow component matches snapshot', () => {
  const component = renderer.create(
    <TaskRow task={MockTasks[0]}></TaskRow>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
