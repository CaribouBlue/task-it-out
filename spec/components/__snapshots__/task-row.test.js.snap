// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskRow component matches snapshot 1`] = `
<tr>
  <td>
    task1
  </td>
  <td>
    Sat Sep 19 2020
  </td>
  <td>
    open
  </td>
</tr>
`;
